build/psgallery-release.yml

```diff
@@ -10,6 +10,39 @@ parameters:
     displayName: 'Prerelease string (ex. -alpha, -alpha1, -BETA, -update20171020) or none'
     type: 'string'
     default: 'none'
+  - name: 'UnitTests'
+    type: object
+    default: 
+    - name: ActiveDirectory
```

Comment:
Maybe we can generate this list based on the test files, that way, we only have to pass in the test folder name to the `run-pester-tests.yml`.

Reply:
I had the same idea initially but after some searching found that apparently this is not possible, see for example https://stackoverflow.com/questions/73332856/azure-pipelines-iterate-through-files

Reply:
Hmm, that StackOverflow issue talks about you can't use absolute paths, if I'm correct. We've done in the past these kinds of things before, if we use `$(Build.SourcesDirectory)` it would be alright. 
Just thinking that a hard-coded list would be rather troublesome to maintain, especially if things change.

Reply:
Will do some more research into this 👍🏻